frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -1321,13 +1331,29 @@ const DetailShortcut = forwardRef((_props, ref) => (
 
 DetailShortcut.displayName = "DetailShortcut";
 
-function ColumnShortcut({ height, onClick, isEditable }) {
-  if (!isEditable) {
+const COLUMN_SHORTCUT_PADDING = 4;
+
+function ColumnShortcut({ height, pageWidth, totalWidth, onClick }) {
+  if (!totalWidth) {
     return null;
   }
 
+  const width = height + COLUMN_SHORTCUT_PADDING;
```

Comment:
I'm curious why do we calculate width based on height? is it kind of design requirement?

Reply:
It's square a button with some padding on the left and right.